### PR TITLE
Documentation

### LINKED ISSUE #22: README
### Description of feature

1. Introduction
2. Pipeline Summary
3. Quick Start
4. Documentation
5. Credits
6. Contributions and Support
7. Citations

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -6,20 +6,31 @@ This document describes the output produced by the pipeline.
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
+- [DeepVaraint](#deepvariant) - VCF and GVCF GZ files generated by DeepVariant
```
Comment (muffato) [inline docs/output.md:13]:
```suggestion
- [DeepVariant](#deepvariant) - VCF and GVCF GZ files generated by DeepVariant
```

File: docs/usage.md [hunk 2]
```diff
@@ -19,34 +17,29 @@ You will need to create a samplesheet with information about the samples you wou
...
+| Column      | Description                                                                                                                                                                            |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample`    | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `datatype`  | Sequencing data type, currently only support Pacbio data.                                                                                                                              |
+| `datafile`  | The localtion for either BAM or CRAM file.                                                                                                                                             |
```
Comment (muffato) [inline docs/usage.md:41]:
```suggestion
| `datafile`  | The location for either BAM or CRAM file.                                                                                                                                             |
```

File: docs/usage.md [hunk 3]
```diff
@@ -19,34 +17,29 @@ You will need to create a samplesheet with information about the samples you wou
...
+| `indexfile` | The localtion for BAM/CRAM index file, BAI, CSI or CRAI file.                                                                                                                          |
```
Comment (muffato) [inline docs/usage.md:42]:
```suggestion
| `indexfile` | The location for BAM/CRAM index file, BAI, CSI or CRAI file.                                                                                                                          |
```

File: README.md [hunk 4]
```diff
@@ -43,30 +48,34 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
-<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
-<!-- If you use  sanger-tol/variantcalling for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
+If you use sanger-tol/variantcalling for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX)
+
+Please also cite DeepVariant:
 
-<!-- TODO nf-core: Add bibliography of tools and data used in your pipeline -->
+> [A universal SNP and small-indel variant caller using deep neural networks. _Nature Biotechnology_ 36, 983–987 (2018).](https://rdcu.be/7Dhl) <br/>
+> Ryan Poplin, Pi-Chuan Chang, David Alexander, Scott Schwartz, Thomas Colthurst, Alexander Ku, Dan Newburger, Jojo Dijamco, Nam Nguyen, Pegah T. Afshar, Sam S. Gross, Lizzie Dorfman, Cory Y. McLean, and Mark A. DePristo.<br/>
+> doi: https://doi.org/10.1038/nbt.4235
```
[the remark this answers is not in the document]
Comment (gq1) [reply README.md:78]: Because this pipeline is about DeepVariant.

File: CITATIONS.md
[the remark this answers is not in the document]
Comment (gq1) [reply CITATIONS.md:1]: This format I copied from DeepVariant pages. I only change the one in the CITATIONS.md to follow the format of others.

**Comment by priyanka-surana** [comment]
@muffato The pipeline is release ready once this PR is merged. Can you do the review? Thanks.

**Review by muffato — COMMENTED** [review]
DeepVariant also creates tabix indexes for the (g)VCF files. Can they be outputted / published too ?

Everything else is fine by me

**Comment by priyanka-surana** [comment]
> DeepVariant also creates tabix indexes for the (g)VCF files. Can they be outputted / published too ?
They are not merged yet like the vcf files themselves. We can add that as a future feature.

**Review by muffato — APPROVED** [review]

**Pull request merged**